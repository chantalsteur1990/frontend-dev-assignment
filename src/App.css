.app {
  margin: 6px;
}

@media screen and (min-width: 1024px) {
  .app {
    width: 1024px;
    margin: 6px auto;
  }
}