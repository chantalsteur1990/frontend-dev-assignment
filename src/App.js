import React, { Component } from 'react';
import Search from './components/search/search';
import './App.css';

class App extends Component {
  render() {
    return (
      <div className="app">
          <Search name="search" placeholder="Zoeken" handleChange={() => {}} ariaLabel="zoeken" />
      </div>
    );
  }
}

export default App;
