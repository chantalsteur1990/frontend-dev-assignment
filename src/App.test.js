import React from 'react';
import { shallow, configure } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';
import App from './App';

configure({ adapter: new Adapter() });

describe('App', () => {
  it('should render (snapshot)', () => {
    const component = shallow(<App />);
    expect(shallowToJson(component)).toMatchSnapshot();
  });
});
