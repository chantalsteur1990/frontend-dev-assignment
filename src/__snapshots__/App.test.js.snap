// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render (snapshot) 1`] = `
<div
  className="app"
>
  <SearchContainer
    ariaLabelClear="annuleren"
    ariaLabelInput="zoeken"
    ariaLabelSubmit="zoeken"
    handleSearch={[Function]}
    handleSuggestions={[Function]}
    name="search"
    placeholder="Zoeken"
  />
</div>
`;
